.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+   "presets": [ "react", "es2015" ],
+   "plugins": [
+      "transform-es2015-modules-commonjs",
+      "transform-react-constant-elements",
```

Comment:
I don't think you want to enable `transform-react-constant-elements` in development.
From what I know it's meant for production only.